nebula/ui/components/MZHelpSheet.qml

```diff
@@ -82,7 +81,8 @@ MZBottomSheet {
                         id: icon
                         anchors.centerIn: parent
 
-                        sourceSize.width: MZTheme.theme.iconSize
+                        source: "qrc:/nebula/resources/tip-filled.svg"
```

Comment:
Was the iconSource property alias removed because MZHelpSheet is expected to always use tip-filled.svg?